systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -295,57 +299,57 @@ public KafkaBuilder defaultKafka(String name, int kafkaReplicas, int zookeeperRe
         String uOImage = StUtils.changeOrgAndTag(getImageValueFromCO("STRIMZI_DEFAULT_USER_OPERATOR_IMAGE"));
 
         return new KafkaBuilder()
-                    .withMetadata(new ObjectMetaBuilder().withName(name).withNamespace(client().getNamespace()).build())
-                    .withNewSpec()
-                        .withNewKafka()
-                            .withVersion(KAFKA_VERSION)
-                            .withReplicas(kafkaReplicas)
-                            .withNewEphemeralStorage().endEphemeralStorage()
-                            .addToConfig("offsets.topic.replication.factor", Math.min(kafkaReplicas, 3))
-                            .addToConfig("transaction.state.log.min.isr", Math.min(kafkaReplicas, 2))
-                            .addToConfig("transaction.state.log.replication.factor", Math.min(kafkaReplicas, 3))
-                            .withNewListeners()
-                                .withNewPlain().endPlain()
-                                .withNewTls().endTls()
-                            .endListeners()
-                            .withNewReadinessProbe()
-                                .withInitialDelaySeconds(15)
-                                .withTimeoutSeconds(5)
-                            .endReadinessProbe()
-                            .withNewLivenessProbe()
-                                .withInitialDelaySeconds(15)
-                                .withTimeoutSeconds(5)
-                            .endLivenessProbe()
-                            .withResources(new ResourceRequirementsBuilder()
-                                .addToRequests("memory", new Quantity("1G")).build())
-                            .withMetrics(new HashMap<>())
-                            .withNewJvmOptions()
-                                .withGcLoggingEnabled(false)
-                            .endJvmOptions()
-                        .endKafka()
-                        .withNewZookeeper()
-                            .withReplicas(zookeeperReplicas)
-                            .withResources(new ResourceRequirementsBuilder()
-                                .addToRequests("memory", new Quantity("1G")).build())
-                            .withMetrics(new HashMap<>())
-                            .withNewReadinessProbe()
+                .withMetadata(new ObjectMetaBuilder().withName(name).withNamespace(client().getNamespace()).build())
+                .withNewSpec()
```

Comment:
I know this is annoying (I suspect IDEA does this), but we should keep the formatted style.